src/views/topics/TopicDetailView.vue

```diff
@@ -85,11 +85,25 @@ const breadcrumbLinks = computed(() => {
   return breadcrumbs
 })
 
-const tabTitles = [
-  { title: 'Données', tabId: 'tab-0', panelId: 'tab-content-0' },
-  { title: 'Discussions', tabId: 'tab-1', panelId: 'tab-content-1' },
-  { title: 'Réutilisations', tabId: 'tab-2', panelId: 'tab-content-2' }
-]
+const tabTitles: { title: string; tabId: string; panelId: string }[] = []
+if (pageConf.datasets.display === true) {
+  tabTitles.push({ title: 'Données', tabId: 'tab-0', panelId: 'tab-content-0' })
```

Comment:
The int indices in `tabId` and `panelId` don't make as much sense now. And could even be confusing since we could have "tab-2" at index 0.

Could we rename them to the actual contents? I.e "tab-0" -> "tab-datasets" etc.

Reply:
Sure